types/google-maps-web/tslint.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-qualifier": false,
+        "strict-export-declare-modifiers": false,
+        "use-default-type-parameter": false,
+        "no-mergeable-namespace": false,
+        "ban-types": false,
```

Comment:
what are these needed for? Is it possible to change the generation code to avoid needing them?

Reply:
I'll take a look to see if there is any low hanging fruit. fyi this is using our internally maintained version of https://github.com/angular/clutz

Reply:
`strict-export-declare-modifiers`: **fixed**

`use-default-type-parameter`,
`no-mergeable-namespace`: not worth the effort to fix at this time as these rules are about being overly verbose in the definitions

`ban-types`: **blocked**, this is for `Function` in some callback handlers